pom.xml

```diff
@@ -56,6 +56,11 @@
   </build>
 
   <dependencies>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>jackson2-api</artifactId>
```

Comment:
From a quick look at `mvn dependency:tree`, it looks like the Jackson libraries are picked up through the plugin, which is what you want, but I'm not 100% sure why they come through this dependency and not github-api, since Jackson is a one-level deep transitive dependency in both cases, so maybe it's just a matter of ordering or something. I would explicitly exclude Jackson from the `github-api` dependency in case an update to that library might change the behavior used by Maven.